### DISCUSSION TITLE
Feature request: support for [x]it!

### DISCUSSION THREAD

**Discussion by igor-arkhipov** [notaapp/talk#182] [Feature Requests]
[x]it! is a plain-text file format for todos and check lists. 
Seems good, but now in draft/RFC status.

[Demo](https://xit.jotaen.net/) 
[Specification](https://github.com/jotaen/xit/blob/main/Specification.md)

**Comment by astoilkov** [comment]
I also saw this on Hacker News. I liked it. However, it's too early to jump to implementations. I would like to wait more and see if it's going to get some traction.

**Comment by Auratelience** [comment]
Is there any documentation for extensions? Cause it seems like a pretty simple extension to make.

**Reply by astoilkov** [reply]
We have extensions (see https://nota.md/docs → Extensions section) but they won't allow you to alter the experience inside of Nota — no way to highlight text and autocomplete text.
